src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -771,6 +778,14 @@ static inline int MPIDI_NM_mpi_compare_and_swap(const void *origin_addr,
     return mpi_errno;
   fn_fail:
     goto fn_exit;
+  am_fallback:
+    if (MPIDI_CH4U_WIN(win, info_args).accumulate_ordering & (MPIDI_CH4I_ACCU_ORDER_WAW | MPIDI_CH4I_ACCU_ORDER_WAR)) {
+        /* Wait for OFI acc to complete.
+         * For now, there is no FI flag to track atomic only ops, we use RMA level cntr. */
```

Comment:
Should this also include `MPIDI_CH4I_ACCU_ORDER_RAW` ? And please also replace `acc` with `cas` in the comment.

Reply:
We only have counters for all `acc` ops, here `cas` belongs `acc`.

Reply:
I will make comment more clear there, and add `MPIDI_CH4I_ACCU_ORDER_RAW`. Also, I will add `void` for `max_size` in `comp_and_swap` to eliminate `unused variable` warning in default mode.

Reply:
Oh, my bad, it is `ofi` level fence, so the comment below has explained that there is no cas only cntr.